dask/blockwise.py

```diff
@@ -37,7 +37,10 @@ def index_subs(ind, substitution):
         return tuple([substitution.get(c, c) for c in ind])
 
 
-def blockwise_token(i, prefix="_"):
+blockwise_default_prefix = "__default__"
```

Comment:
is it possible to change to __dask_default__?

Reply:
Perhaps a label that makes it more obvious that it is an abstract "blockwise" label would be even better (like `"__blockwise__"`).  Though,  I certainly dont feel strongly about this.